distgen/doc.go

```diff
@@ -56,5 +56,17 @@ global variables to generate. (See an example of a JSON file at
   package dist
   //go:generate go run github.com/chef/go-libs/distgen global.go dist https://example.com/path/to/glob_dist.json
 
+Using an Environment Variable
+
+The usage of an environmental variable allows for better flexibility when dealing with CI systems.
+
+As we saw in the previous example, the 3rd argument that is passed to `go run` is an URI, which is used to override the default JSON file for `distgen`.
+
+The `DIST_FILE` environment performs the same functionality by overloading the default JSON file.
```

Comment:
Go docs doesn’t have a concept of ` (back ticks), so it is better to substitute them for double quotes or no surrounding things. I know it feels weird but it renders and reads better without them.